python/openai/openai_frontend/main.py

```diff
@@ -104,6 +129,17 @@ def main():
         log_error=True,
     ).start(wait_until_ready=True)
 
+    http_options, http_service = None, None
+    grpc_options, grpc_service = None, None
+    if "tritonfrontend" in sys.modules:
```

Comment:
defer import to here?

Reply:
Does it work as intended by moving the import there?
```
Python 3.10.12 (main, Jul 29 2024, 16:56:48) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> "numpy" in sys.modules
False
>>> import numpy
>>> "numpy" in sys.modules
True
>>>
```

Reply:
I was planning on replacing the logic to look something like this:
```python3
http_service, grpc_service = None, None
try:
     from tritonfrontend import KServeGrpc, KServeHttp
     http_service, grpc_service = start_kserve_frontends(server, args)
except ModuleNotFoundError:
     print(...)
     
# OpenAI Frontend logic
# Blocking call until killed or interrupted with SIGINT
frontend.start()

if http_service:
    http_service.stop()
if grpc_service:
    grpc_service.stop()
```